stable/selenium/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: selenium
-version: 0.3.1
-appVersion: 3.11.0
+version: 0.3.2
```

Comment:
Please bump the minor version instead

Reply:
@rjkernick I updated the minor version as requested. I think there is a lack of definition concerning the usage of Semantic Versioning ( previous updates of Selenium version have been "marked" via patch version in Chart.yaml )